### PR TITLE
Preview

### PR DESCRIPTION
**Pull request by edwinmp**
- Add icon
- Remove desktop preview
- Web modeler preview
- Move from css to scss
- Update project dependencies

### CODE REVIEW

File: src/components/CarouselContainer.ts [hunk 1]
```diff
@@ -3,8 +3,11 @@ import { Carousel, Image } from "./Carousel";
 import { Alert } from "./Alert";
 import { UrlHelper } from "../UrlHelper";
 
-interface CarouselContainerProps {
+interface WrapperProps {
     mxObject: mendix.lib.MxObject;
```
Comment (Andries-Smit) [inline src/components/CarouselContainer.ts:7]: missing
```
    class?: string;
    mxform: mxui.lib.form._FormBase;
    mxObject?: mendix.lib.MxObject;
    style?: string;
```

File: src/components/CarouselContainer.ts [hunk 2]
```diff
@@ -196,3 +199,5 @@ export default class CarouselContainer extends Component<CarouselContainerProps,
         return context;
     }
 }
+
+export { CarouselContainer as default, CarouselContainerProps };
```
Comment (Andries-Smit) [inline src/components/CarouselContainer.ts:203]: R&D prefers the export inline, not at the bottom

File: src/ui/Carousel.scss [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    }
+
+    @mixin on-hover-focus {
+        color: #fff;
+        text-decoration: none;
+        filter: alpha(opacity=90);
+        outline: none;
+        opacity: .9;
+    }
+    &:focus {
```
Comment (Andries-Smit) [inline src/ui/Carousel.scss]: missing newline

File: typings/jasmine.d.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
+// tslint:disable no-namespace
 declare namespace jasmine {
     interface Any {
         asymmetricMatch(other: any): boolean;
     }
 
-    interface MatchersUtil {}
-    export var matchersUtil: MatchersUtil;
+    export const matchersUtil: MatchersUtil;
```
Comment (Andries-Smit) [inline typings/jasmine.d.ts:7]: 👍

File: src/Carousel.webmodeler.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import { Component, createElement } from "react";
+
+import { Carousel, Image } from "./components/Carousel";
+import CarouselContainer, { CarouselContainerProps } from "./components/CarouselContainer";
+import { Alert } from "./components/Alert";
+
+// tslint:disable-next-line
+const image = require("base64-image-loader!./img/Preview.jpg");
```
Comment (Andries-Smit) [inline src/Carousel.webmodeler.ts:8]: Can you do this inside the webpack config?

File: src/components/CarouselContainer.ts (hunk 1 above)
Comment (edwinmp) [reply src/components/CarouselContainer.ts:7]: Done.

File: src/components/CarouselContainer.ts (hunk 2 above)
Comment (edwinmp) [reply src/components/CarouselContainer.ts:203]: Done.

File: src/Carousel.webmodeler.ts (hunk 5 above)
Comment (edwinmp) [reply src/Carousel.webmodeler.ts:8]: Done.

**Pull request merged**